browser/src/map/Map.js

```diff
@@ -154,6 +154,11 @@ L.Map = L.Evented.extend({
 		this._progressBar = L.progressOverlay(new L.point(150, 25));
 
 		this._debug = new L.DebugManager(this);
+		this.on('docloaded', function() {
```

Comment:
if possible may be nicer to hook into an existing handler rather than having the "my timeout is bigger than yours" approach etc. Then we know what order handlers are executed in and so on; but not read carefully =)